server/src/test/java/org/opensearch/index/seqno/GlobalCheckpointSyncActionTests.java

```diff
@@ -158,14 +159,14 @@ public void testTranslogSyncAfterGlobalCheckpointSync() throws Exception {
     public void testGetReplicationModeWithRemoteTranslog() {
```

Comment:
We would also need to have correct condition instead of `indexShard.isRemoteTranslogEnabled() == false` - 
https://github.com/opensearch-project/OpenSearch/blob/8ad0dc09d3e852d74cd7eac65882ea7674a282d9/server/src/main/java/org/opensearch/index/seqno/GlobalCheckpointSyncAction.java#L135-L140
Lets double check other places where the same change needs to be done.